src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -86,7 +90,35 @@ public class PublicationService : IPublicationService
                 .OnSuccess(_userService.CheckCanViewPublication)
                 .OnSuccess(publication => _mapper.Map<PublicationViewModel>(publication));
         }
-        
+
+        public async Task<Either<ActionResult, bool>> UpdatePublicationMethodologyAsync(Guid publicationId,
+            UpdatePublicationMethodologyViewModel methodology)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                //.OnSuccess(_userService.CheckCanUpdatePublicationMethodology)
```

Comment:
This line looks like it's redundant now

Reply:
Placeholder for methodology permission work